config.json

```diff
@@ -72,6 +72,12 @@
       "values": [],
       "defaultValue": "/flp:v=normal"
     },
+    "StripSymbols": {
```

Comment:
That was my intention.  If you're asking why I didn't include that change in this PR then... I don't know why I was mentally separating the two.  I can include it.